src/client/src/rt-storybook/Story/Story.tsx

```diff
@@ -66,10 +60,10 @@ const StyledStory = styled('div')`
   display: flex;
   flex-direction: column;
   align-items: stretch;
-  background-color: ${({ theme: { background } }) => background.backgroundPrimary};
-  color: ${({ theme }) => theme.text.primary};
-  transition: background-color ${({ theme }) => theme.animationSpeed.normal}ms,
-    color ${({ theme }) => theme.animationSpeed.normal}ms;
+  background-color: ${p => p.theme.shell.backgroundColor};
+  color: ${p => p.theme.shell.textColor};
```

Comment:
`theme.textColor` to follow what we're doing in other components

Reply:
more characters, more problems.